docs/README.md

```diff
@@ -470,6 +472,83 @@ None : This hooks takes no parameters.
 
 </br>
 
+## 👆 useClickOutside
```

Comment:
@imevanc Please add a simple example, just include the demonstration of the hook, nothing more